libraries/standard/mqtt/utest/mqtt_utest.c

```diff
@@ -1589,6 +1592,7 @@ void test_MQTT_ProcessLoop_handleIncomingAck_Error_Paths( void )
 
     setupTransportInterface( &transport );
     setupCallbacks( &callbacks );
+    setupNetworkBuffer( &networkBuffer );
```

Comment:
This was the _fix_ for the `test_MQTT_ProcessLoop_handleIncomingAck_Error_Paths` test